source/MRMesh/MRVDBConversions.cpp

```diff
@@ -328,22 +337,28 @@ Expected<VoxelsVolume<std::vector<T>>, std::string> vdbVolumeToSimpleVolumeImpl(
         auto& accessor = accessorPerThread.local();
         auto coord = indexer.toPos( VoxelId( i ) );
         float value = accessor.getValue( openvdb::Coord( coord.x + org.x, coord.y + org.y, coord.z + org.z ) );
-        if constexpr ( isFloat )
+        if constexpr ( isFloat && !Norm )
             res.data[i] = T( value );
         else
-            res.data[i] = T( std::clamp( ( value - vdbVolume.min ) * k, 0.0f, oMax ) );
+            res.data[i] = T( std::clamp( ( value - vdbVolume.min ) * k + oMin, oMin, oMax ) );
```

Comment:
`oMin` is zero here. And if it weren't, the formula for `k` should be corrected, too: `(oMax - oMin) / ( vdbVolume.max - vdbVolume.min )`